Documentation related to Instance lease feature (#492)
apache/cloudstack-documentation: source/adminguide/service_offerings.rst

         -  **Disk Offering Strictness**: This flag defines the strictness of the disk offering association 
            with the compute offering. When set to true, overriding of disk offering is not allowed on deploy instance
            and change disk offering is not allowed for the ROOT disk
      

>       -  **Enable Lease**: When this flag is enabled, compute offering is created with lease related metadata.

Reviewer: Should we add here that, this is only available when the feature is enabled via global setting?

Author: Note is included at the end in this section. Should we move it in beginning?

![image](https://github.com/user-attachments/assets/524679d7-04e6-411f-bdcf-b83de7ff41d3)

         -  **Disk Offering Strictness**: This flag defines the strictness of the disk offering association 
            with the compute offering. When set to true, overriding of disk offering is not allowed on deploy instance
            and change disk offering is not allowed for the ROOT disk
      

>    -  **Enable Lease**: When this flag is enabled, Compute Offering is created with 'Instance Lease' feature enabled. 

Reviewer 2: check the indentation here

>       In CloudStack, a lease sets a specific time duration after which a chosen lease action action, such as stopping or destroying the instance, will take place. 

Reviewer 2: ```suggestion
      In CloudStack, a lease for an instance sets a specific time duration (in days) after which a chosen lease action, such as STOP or DESTROY will be performed.
```

         -  **Disk Offering Strictness**: This flag defines the strictness of the disk offering association 
            with the compute offering. When set to true, overriding of disk offering is not allowed on deploy instance
            and change disk offering is not allowed for the ROOT disk
      

>    -  **Enable Lease**: When this flag is enabled, compute offering is created with lease related metadata.

Reviewer 3: When this flag is enabled, Compute Offering is created with 'Instance Lease' feature enabled. In CloudStack, a lease sets a specific time duration after which a chosen lease action action, such as stopping or destroying the instance, will take place. These lease settings are defined in the Compute Offering and are automatically applied to any Instance created using it.

      In CloudStack, a lease represents the specific duration for which an instance is allocated.
      The user rents these resources for the duration of the lease. Once the lease period expires, instance may be stopped or destroyed.
      Lease information is inherited from compute offering and gets associated to Instance.
      
      .. note:: The global configuration ``instance.lease.enabled`` should be configured as true to create compute offering with lease.

         ``instance.lease.enabled``: Indicates whether Instance Lease feature is enabled or not. Default is **false**
         For more information, see `“Setting Global Configuration Parameters”
         <../installguide/configuration.html#setting-global-configuration-parameters>`_.

      When the flag is enabled

>          -  **Lease Duration (in days)**: Creates a compute offering with Lease duration. Instance created from this compute offering will inherit lease duration by default.

Reviewer 3: **Lease Duration (in days)**: Sets the lease duration. An instance created using this compute offering will inherit the lease duration by default.
